samples/apps/copilot-chat-app/webapi/CopilotChatWebApi.csproj

```diff
@@ -62,11 +62,17 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+
+    <PackageReference Include="Tesseract" Version="5.2.0" />
```

Comment:
Have you considered Azure Cognitive Service for OCR: https://azure.microsoft.com/en-us/products/cognitive-services/vision-services/?

Reply:
I would be happy to do a subsequent PR that allows for integration with Cognitive Services.  Tesseract is open source and free, so I figured I'd start there.  But similar to how we allow swapping in Vector DBs like Qdrant & Cognitive Search, I'm happy to do the same for OCR capabilities!

Reply:
I think this is reasonable. Would be nice to add an option for CogServices support later, but for now this is great! Thanks for all your work here!